README.md

```diff
@@ -74,7 +74,7 @@ There are no major restrictions imposed on software submitted to SailfishOS:Chum
 Although one limitation is that submitted software should not interfere with any software distributed by Jolla as a part of Sailfish&nbsp;OS, which also means that updating libraries distributed per Jolla's repositories must be avoided.
 Note that Jolla's repositories provide far more software packages than the ones installed by default.
 
-All kinds of software for Sailfish&nbsp;OS are welcome at SailfishOS:Chum, for example, actively developed SFOS applications, discontinued SFOS applications, tools and libraries missing on SFOS as distributed by Jolla.
+All kinds of software for Sailfish&nbsp;OS are welcome at SailfishOS:Chum, for example, actively maintained software for SFOS, abandoned software for SFOS, and extant tools and libraries which are missing on SFOS as distributed by Jolla.
```

Comment:
I had to look up the meaning of extant!

Reply:
Seriously!?!  Oops, wow, well, it is shorter and supposedly "higher level language" than "existing", which can be ambiguous (depends on the context).  Would you prefer "existing" over "extant"?
I used "third party" further down, which IMO is not perfect either.

Reply:
Or current?, or just "...tools and libraries... ?  We could keep extant, as you say, its well defined, but if a native like myself had to look it up, it may be confusing!

Reply:
> Or current?, or just "...tools and libraries... ?

IMO "current" does not really fit and leaving it out does also not suit this well, because both do not express that these "tools and libraries" are maintained elsewhere and are not originally written for SailfishOS.  In short, that an "upstream repo" exists, which is not under control of the submitter (thus I used "third party" further down).

> We could keep extant, as you say, its well defined, but if a native like myself had to look it up, it may be confusing!

While I try refrain from using "over-the-top language and grammar", because that does create a barrier for people to comprehend the content, and one may perceive "extant" being close to this realm, I also try hard to avoid ambiguities, colloquialisms, using deliberately simple language (where "do" and "get" are the ~~ubiquitous~~ omnipresent verbs, but also the meaning becomes hard to grasp) etc.  Hence I like "extant" and prefer to keep it, because it is concise.

Writing documentation and specifications for an international audience always needs to balance expressiveness against simplicity.  While I tend to prefer expressiveness, I deliberately wrote "~~ubiquitous~~ omnipresent" above, because they mean exactly the same, but I perceive "ubiquitous" as a modern day buzzword and "over-the-top language".

Reply:
please drop "extant". It doesn't help that we have to look it up to read README

Reply:
Done, see [commit d73de84](https://github.com/sailfishos-chum/main/pull/68/commits/d73de84f1c8a69b3d394ed68c52a83ee2cbbba47).

Reply:
Thank you! @piggz - would you finalize this PR? It sounded like you reviewed it already